CHANGELOG.md

```diff
@@ -14,9 +14,36 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 
 #### Logic Database
 
+- Added: New trick, Cross Bomb Launch.
+
+##### Burenia
+
+- Added: Crossing the gap in Underneath Drogyga with Cross Bomb Launch (Advanced).
+- Added: Reaching the Missile Tank Pickup in Main Hub Tower Top using Cross Bomb Launch (Intermediate).
```

Comment:
For some reason i feel like these should be swapped considering the second has vertical distance required as well.